app/models/energy/ScheduleFile.py

```diff
@@ -0,0 +1,57 @@
+"""Schedule File Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+from app.models.energy.ScheduleBase import UnitType
+
+
+class ScheduleFile(BaseModel):
+    """A list of values used to create a .csv file for the entire year or a period of
+  analysis within the year."""
+
+    type: Enum('ScheduleFile', {'type': 'ScheduleFile'})
+
+    name: str = Schema(
```

Comment:
This is just a name?  Not a filename?

Reply:
I was imagining that the file name will be generated from the schedule name in the energy-model-measure since the current plan is to have the measure write all of the values into the file.

Reply:
This is no longer relevant since we decided to go with ScheduleInterval.

Reply:
This isn't relevant anymore now that we have decided to go with fixed interval schedules.